src/Donations/PaymentMethods/PaymentRequestCustomButton.tsx

```diff
@@ -182,22 +181,7 @@ export const PaymentRequestCustomButton = ({
               <div className="separator-text mb-10">{t("or")}</div>
             )}
           </div>
-        ) : (
-          <div className="w-100">
-            <button
-              onClick={() => paymentRequest.show()}
-              className={`donate-now ${
-                isPaymentPage ? "donate-small" : "primary-button mb-10 w-100"
-              }`}
-              style={{ border: "none" }}
-            >
-              {isPaymentPage ? "" : t("donateNow")} <BrowserPayIcon />
```

Comment:
I think the resource `donateNow` is also not used any more after removing this, but I am not sure if we want to keep unused translations.

Reply:
I've removed the unused translations - better to keep things clean.